README.md

```diff
@@ -12,8 +12,16 @@ like Excel or simply storing a backup of the source data set.
 ### Install and Run
 
 First, make sure Python 3 is installed on your system or follow these
-installation instructions for [Mac](python-mac) or
-[Ubuntu](python-ubuntu).
+installation instructions for [Mac] or
+[Ubuntu].
```

Comment:
Whoa. Was not aware of this Markdown feature. Is this GFM or Markdown proper? (Linking to a named ref without the trailing `[]`).

Reply:
Actually is just proper Markdown. Check this out https://github.com/adam-p/markdown-here/wiki/Markdown-Here-Cheatsheet#links